core/src/main/java/org/apache/carbondata/core/datamap/AbstractDataMapJob.java

```diff
@@ -30,4 +30,8 @@
   @Override public void execute(CarbonTable carbonTable,
       FileInputFormat<Void, BlockletDataMapIndexWrapper> format) {
   }
+
+  @Override public Long executeCountJob(DistributableDataMapFormat dataMapFormat) {
+    return null;
```

Comment:
retrun 0 instead of null

Reply:
removed the implementation in AbstractDataMapJob.java, now all the extending classes should give their own implementations